docs/source/guide/prompts_overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Prompts overview
+short: Prompts overview
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 225
+meta_title: Prompts overview
+meta_description: An overview of the Prompts feature in the HumanSignal platform
+section: Prompts
+date: 2024-05-15 14:30:14
+---
+
+
+Use Prompts to evaluate and refine your LLM prompts, and then generate predictions to automate your labeling process. 
+
+All you need to get started is an OpenAI API key and a dataset with some ground truth annotations. 
+
+With Prompts, you can:
+
+* Drastically increase the speed and efficiency of annotations, transforming subject matter experts (SMEs) into highly productive data scientists while reducing the dependency on non-SME manual annotators.
+* Enhance annotation throughput and accuracy, making the process faster and more scalable. 
+* Empower users to harness the full potential of AI-driven text labeling, setting a new standard for efficiency and innovation in data labeling.
+
+## Auto-labeling with Prompts
+ 
+Prompts allows you to leverage LLMs to swiftly generate accurate predictions, enabling instant labeling of thousands of tasks. 
+
+By utilizing AI to handle the bulk of the annotation work, you can significantly enhance the efficiency and speed of your data labeling workflows. This is particularly valuable when dealing with large datasets that require consistent and accurate labeling. Automating this process reduces the reliance on manual annotators, which not only cuts down on labor costs but also minimizes human errors and biases. With AI's ability to learn from the provided ground truth annotations, you can maintain a high level of accuracy and consistency across the dataset, ensuring high-quality labeled data for training machine learning models.
+
+
+
+## Prompt evaluation and fine-tuning
+
+!!! info Tip
+    While you can use Prompts to do basic prompt evaluation and fine-tuning, we recommend using [Evals](evals_overview), which has been specifically designed for this purpose and has much more robust analytics. 
```

Comment:
i wouldn't say Evals is designed for finetuning, but i'd agree it's better for prompt evaluation. As such, i'd suggest perhaps emphasize Prompts as the interface for building prompts, and evals as the tool to evaluate - maybe wording to something along the lines of -
While you can use Prompts to conduct prompt fine-tuning and basic evaluations, we recommend using Evals for deeper analyses on LLM prompt performance and evaluations.

Reply:
Tentatively removing that tip until I can be sure that Evals docs will need to be out